src/components/TitleHashButton.tsx

```diff
@@ -109,7 +109,7 @@ function HashButton({hash}: {hash: string}) {
 
 function Name({address}: {address: string}) {
   const theme = useTheme();
-  const name = useGetNameFromAddress(address);
+  const name = useGetNameFromAddress(address, true);
```

Comment:
why do we cache here but not in `HashButton.tsx` (https://github.com/aptos-labs/explorer/blob/36b777ef8fe80a3eee5cca0282b58ff6854a36e5/src/components/HashButton.tsx#L92)

Reply:
we try to get name of `HashButton.tsx` whenever there's an address, (e.g. on an account detail page, we try to get the name for sender / receiver of every tx), im concerned this will save too much stuff into the cache, plus i think only the account owner's name is the most commonly appeared one across the page, so i only cache the owner